core-plugins/docs/AzureBlobStore-batchsource.md

```diff
@@ -16,7 +16,7 @@ Properties
 ----------
 **referenceName:** This will be used to uniquely identify this source for lineage, annotating metadata, etc.
 
-**path:** The path on Microsoft Azure Blob Storage to use as input.
+**path:** The path on Microsoft Azure Blob Storage to use as input. Also, the path uses Globbing to read files.
```

Comment:
The path on Microsoft Azure Blob Storage to use as input. The path uses filename expansion (globbing) to read files.